Application/mobile/app_code/lib/components/parameter.dart

```diff
@@ -14,6 +14,14 @@ enum DIVIDERPLACE {
   bottom,
 }
 
+/// This class is used to create a divider that redirects to another page.
+/// It is used in the settings page.
```

Comment:
What if we (want to) use it on another page ?

Reply:
you can, it's a component